tests/common.rs

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I think those tests do not intend to be against the module commonly used. I think `auxiliary.rs` is a more appropriate name.